### ISSUE TITLE
CF needs migrations 

### ISSUE THREAD

**Issue by LindsayYoung** [18F/openFEC#528]
The API in cloud foundry needs to run migration to get the tables it needs. 

example:
`http://fec-api-dev.cf.18f.us/candidate/H4UT04052`
produces:
```
sqlalchemy.exc.ProgrammingError: (ProgrammingError) column ofec_candidate_detail_vw.candidate_expire_date does not exist
LINE 2: FROM (SELECT ofec_candidate_detail_vw.candidate_expire_date ...
```

I tried launching the app with a shell script that runs the migration and then the api, but the migrations seemed to error out. Maybe it is permissions? 

Task:
- [ ] get `python manage.py refresh_db` to run successfully on cloudfoundry before starting the app

**Comment by LindsayYoung** [comment]
Here is the branch with the python 3 cloud foundry stuff and a launch script that runs the migrations:
https://github.com/18F/openFEC/tree/cloud_startup

Error from cf:
```
      ERR Traceback (most recent call last):
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1063, in _execute_context
      ERR     context)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/default.py", line 442, in do_execute
      ERR     cursor.execute(statement, parameters)
      ERR psycopg2.ProgrammingError: must be owner of relation ofec_committees_vw
      ERR The above exception was the direct cause of the following exception:
      ERR Traceback (most recent call last):
      ERR   File "manage.py", line 47, in <module>
      ERR     manager.run()
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/flask_script/__init__.py", line 412, in run
      ERR     result = self.handle(sys.argv[0], sys.argv[1:])
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/flask_script/__init__.py", line 383, in handle
      ERR     res = handle(*args, **config)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/flask_script/commands.py", line 216, in __call__
      ERR     return self.run(*args, **kwargs)
      ERR   File "manage.py", line 41, in refresh_db
      ERR     db.engine.execute(sql)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1863, in execute
      ERR     return connection.execute(statement, *multiparams, **params)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 833, in execute
      ERR     return self._execute_text(object, multiparams, params)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 982, in _execute_text
      ERR     statement, parameters
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1070, in _execute_context
      ERR     context)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1271, in _handle_dbapi_exception
      ERR     exc_info
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/util/compat.py", line 188, in raise_from_cause
      ERR     reraise(type(exception), exception, tb=exc_tb, cause=exc_value)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/util/compat.py", line 181, in reraise
      ERR     raise value.with_traceback(tb)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1063, in _execute_context
      ERR     context)
      ERR   File "/app/.heroku/python/lib/python3.4/site-packages/sqlalchemy/engine/default.py", line 442, in do_execute
      ERR     cursor.execute(statement, parameters)
      ERR sqlalchemy.exc.ProgrammingError: (ProgrammingError) must be owner of relation ofec_committees_vw
```

**Comment by arowla** [comment]
There were a few issues here. The app needed to be changed to not use a read-only user. That was changed in the CF ENV vars.

The startup script @LindsayYoung tried wasn't working because the migrations were timing out. I solved this by running the migrations manually through `cf-ssh`. See our devops docs for installing and running [CF-SSH](https://docs.cf.18f.us/getting-started/cf-ssh/).

The steps to run the DB refresh are:

* In a terminal, run `cf env fec-api-stage` and record value of `SQLA_CONN`
* Run `cf-ssh fec-api-stage`
* `export SQLA_CONN=<connection string recorded in first step>`
* `python manage.py refresh_db`
* Wait until finished. Look for any errors, if none, `exit`

I also up-sized the dev RDS instance from 4GB RAM to 15GB RAM to help the DB views build more quickly.
